IMU_Testing/IMU_Testing.cpp

```diff
@@ -0,0 +1,389 @@
+#include "IMU_Testing.h"
+#include <SparkFunMPU9250-DMP.h>
+#define SerialPort SerialUSB
+
+MPU9250_DMP imu;
+
+const signed char IMU_Testing::orientationMatrix[9]={1,0,0,0,1,0,0,0,1};
+unsigned char IMU_Testing::lastOrient = 0;
+
+IMU_Testing::IMU_Testing()
+{}
+
+void IMU_Testing::setupEverything()
+{
+  SerialPort.begin(115200);
```

Comment:
Does commenting them out work as for testing would allow them to be useful?

Reply:
For code style and clarity, it would just best to remove the prints and restructure the code so that any methods with any need for that would just return the value and would get logged somewhere else.

Reply:
Well if we don't need the print____ functions, then I can just get rid of them completely.